client/src/pages/All_Modules/index.js

```diff
@@ -0,0 +1,10 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Hi Kristina, please I want to ask, do we need  "pages" and "all modules" folders outside of the "components" folder. Do we need these two folders?
 Also, is it possible to change the name of the "index" file to something more intuitive, as it can be confusing for others? Additionally, I have seen about using "container" as a class name since it is a commonly used name and may conflict with other CSS. What do you think about these points? :)